exampleLitActions/test/fetch.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import { fetch } from '../src/fetch';
+import * as dotenv from 'dotenv';
+
+// Load environment variables from .env file if present
+dotenv.config();
+
+describe('Lit Action Weather-Based Test', function() {
+  // Increase timeout for API calls and blockchain interactions
+  this.timeout(60000);
+  
+  before(function() {
+    // Skip tests if required environment variables are missing
+    if (!process.env.ETHEREUM_PRIVATE_KEY) {
+      console.warn('⚠️ Tests require ETHEREUM_PRIVATE_KEY environment variable');
+      this.skip();
+    }
+  });
```

Comment:
We shouldn't skip the tests an error should be thrown, and the `getEnv` util functions should catch/handle this